packages/cli/src/commands/test.js

```diff
@@ -86,22 +110,22 @@ export const handler = async ({
   }
 
   if (sides.includes('api')) {
-    args.push('--runInBand')
+    jestArgs.push('--runInBand')
   }
```

Comment:
Unrelated, but this made me wonder if this doesn't now run all the tests in band... making everything slower. I wonder if there's a way to set this up in the configuration for the api side rather than as an argument.

Apparently it's not supported, but maybe this will work:

https://github.com/facebook/jest/issues/3215#issuecomment-660593411

Reply:
Do we do this because we expect the api tests to run in a particular sequence?

Reply:
It's because of scenarios I believe. I don't think named scenarios work as expected anyway tbh